FIAT/__init__.py

```diff
@@ -50,7 +50,6 @@
                       "Discontinuous Raviart-Thomas": DiscontinuousRaviartThomas,
                       "Hermite": CubicHermite,
                       "Lagrange": Lagrange,
-                      "Gauss-Lobatto-Legendre": GaussLobattoLegendre,
```

Comment:
Ah this was a mistake. I think I forgot this line when I was doing a rebase...Sorry about that. Shall I correct it and push?